Application/TelegramBot.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Configuration;
+using Telegram.Bot;
+
+namespace Application
+{
+    public class TelegramBot 
+    {
+        private readonly IConfiguration _configuration;
+        private TelegramBotClient _botClient;
+        public TelegramBot(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        public async Task<TelegramBotClient> GetBot()
+        {
+            _botClient = new TelegramBotClient(_configuration["Token"]);
```

Comment:
if we create client each call, I do not see reason to register TelegramBot as singleton and not scoped

Reply:
Done